src/clients/twitch.ts

```diff
@@ -83,6 +84,38 @@ function registerTwitch(): void
             if(!bridge.targetDiscordChannel)
                 throw new Error('Cannot find Discord channel.');
 
+            /**
+                    'reply-parent-display-name': 'testingaccount__',
+                    'reply-parent-msg-body': 'test',
+                    'reply-parent-msg-id': 'f279b175-7100-4486-bb96-c188ea102bbd',
+                    'reply-parent-user-id': '821973125',
+  '                 'reply-parent-user-login': 'testingaccount__',
+
+                    Those keys will exist on the userState.tags (Map<String, String>) map, if it
+                    is a reply.
+            */
+
+            // if there is a reply parent display name
+            // we know it's a reply.
+            if(userState.tags.get('reply-parent-display-name'))
```

Comment:
There is a bug here upon testing the code!

If a discord user replies to another discord user, the bot will take the sent message, and in turn make it over here. The fix for this is to check if the bot is receiving a message from itself like what's being done in [this line](https://github.com/iSlammedMyKindle/twitchToDiscordBot/pull/12/files#diff-abef7cdea7215ad33afd80fd66b5862a41c341b32d439e9d9c8da5e996e1d231R121)